src/lib.rs

```diff
@@ -3,6 +3,278 @@
 //! It supports multiple indexes, real-time watch with filters, model migration, hot snapshot, and more.
 //!
 //! See [README.md](https://github.com/vincent-herlemont/native_db) for more information.
+//!
+//! # Quick Start
+//!
+//! 1. [Create a model](#create-a-model)
+//! 2. [Create the database with the model](#create-the-database-with-the-model)
+//! 3. [Use a model in the database](#use-a-model-in-the-database)
+//! 4. [Update the model](#update-the-model)
+//! 5. [Use the updated model in the database (migration)](#use-the-updated-model-in-the-database-migration)
+//!
+//! ## Create a model
+//!
+//! > 👉 Unlike the usual database where there is a differentiation between *schema* and *model*, here, as we can directly use Rust types that are serialized in the database, we do not have the concept of *schema*, only that of the *model*.
+//!
+//! Note that the organization of the models it's a best practice but not mandatory, you can organize your models as you want if you prefer.
```

Comment:
grammar: "Note that the organization of the models is only a best practice, not mandatory. You can organize your models as you want."

Reply:
Done